protocol/contracts/amo/interfaces/AMO__IOudtoken.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.8.17;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Origami (interfaces/common/IMintableToken.sol)
```

Comment:
This shouldn't be sitting in the amo dir (and shouldn't have the `AMO__` prefix)

My preference (after working through lots of interface files) is:
`contracts/oud/OudToken.sol`
`contract/interfaces/oud/IOudToken.sol'

Also, it's not origami, so this line needs updating.

Reply:
fixed in [ac5905d](https://github.com/TempleDAO/temple/pull/760/commits/ac5905d0d1b244eee387df8ce2f0d382287efcde)